docs/source/modin/supported/datetime_index_supported.rst

```diff
@@ -0,0 +1,284 @@
+``pd.DatetimeIndex`` supported APIs
```

Comment:
Actually, after a second look, can we limit the list of attributes and methods here to the ones listed in the [pandas docs for DatetimeIndex](https://pandas.pydata.org/docs/reference/api/pandas.DatetimeIndex.html). Many of the currently listed ones here belong to `Index` but not `DatetimeIndex`.

Reply:
done